### DISCUSSION TITLE
add custom widgets

### DISCUSSION THREAD

**Discussion by daveningfr** [end-4/dots-hyprland#843] [Q&A] [answered]
hi im quite new to arch and ags but i want to add a custom widget in the top bar but idk where to start. i searched everywhere but i cannot find the documentation for ags all help is appreciated thanks

**Comment by end-4** [comment, marked as answer]
change stuff in ~/.config/ags/modules/bar/normal
ags wiki: https://aylur.github.io/ags-docs/

**Reply by daveningfr** [reply]
thanks ur so good
